[{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12451069","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12451069","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12451069,"user":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false},"created_at":"2013-01-19t06:05:07z","updated_at":"2013-01-19t06:05:07z","body":"what is the cookie? what auth version? \r\n\r\ni'm having trouble understanding what you're saying, can you provide exact steps to reproduce. this is also a good branch to test on with external storage: #1031 \r\n"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12453065","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12453065","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12453065,"user":{"login":"henrynopo","id":3304600,"avatar_url":"https://gravatar.com/avatar/c5bbcf2196f976a7775b5a7f643c330d?d=https%3a%2f%2fidenticons.github.com%2ffc2614345a8665029e8790245655da84.png&r=x","gravatar_id":"c5bbcf2196f976a7775b5a7f643c330d","url":"https://api.github.com/users/henrynopo","html_url":"https://github.com/henrynopo","followers_url":"https://api.github.com/users/henrynopo/followers","following_url":"https://api.github.com/users/henrynopo/following{/other_user}","gists_url":"https://api.github.com/users/henrynopo/gists{/gist_id}","starred_url":"https://api.github.com/users/henrynopo/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/henrynopo/subscriptions","organizations_url":"https://api.github.com/users/henrynopo/orgs","repos_url":"https://api.github.com/users/henrynopo/repos","events_url":"https://api.github.com/users/henrynopo/events{/privacy}","received_events_url":"https://api.github.com/users/henrynopo/received_events","type":"user","site_admin":false},"created_at":"2013-01-19t10:25:48z","updated_at":"2013-01-19t10:27:17z","body":"i got the same problem as reflected in the owncloud forum. the topic is \"using external storage dropbox gdrive with owncloud 4.5\", refer to : http://forum.owncloud.org/viewtopic.php?f=3&t=3696\r\n\r\nthe issue due to unmounting google drive from external storage are reflected inside, such as by \"topedia\". it seems a lot of people got the same issue too. your post #400 seems similar.\r\n\r\nthe issue unable to mount google drive even for the first time was also reflected. i tried by myself as advised in that topic that by clearing all the cookies of internet explorer the google drive can be mounted into the \"files\" even though access may not be granted successfully.\r\n\r\nfor oauth version, my understanding it that the owncloud 4.5 is using google oauth 1.0 or 2.0 which are recommended by google to be updated to the latest by using api access. when i check the scope of access requested by the app, it seems requesting for the previous google docs instead of the actual google drive.\r\n\r\nanyway, i am not it guy but just curious with the external storage function provided by owncloud. thanks.\r\n"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12784591","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12784591","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12784591,"user":{"login":"torlasz","id":2997536,"avatar_url":"https://gravatar.com/avatar/d4184dba791f5d4ca7701cb4d40e21d2?d=https%3a%2f%2fidenticons.github.com%2fa0b69955c7185e378fc6dd2c04979570.png&r=x","gravatar_id":"d4184dba791f5d4ca7701cb4d40e21d2","url":"https://api.github.com/users/torlasz","html_url":"https://github.com/torlasz","followers_url":"https://api.github.com/users/torlasz/followers","following_url":"https://api.github.com/users/torlasz/following{/other_user}","gists_url":"https://api.github.com/users/torlasz/gists{/gist_id}","starred_url":"https://api.github.com/users/torlasz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/torlasz/subscriptions","organizations_url":"https://api.github.com/users/torlasz/orgs","repos_url":"https://api.github.com/users/torlasz/repos","events_url":"https://api.github.com/users/torlasz/events{/privacy}","received_events_url":"https://api.github.com/users/torlasz/received_events","type":"user","site_admin":false},"created_at":"2013-01-28t14:44:28z","updated_at":"2013-01-28t14:58:01z","body":"i am not able to mount a google drive with a freshly installed 4.5.6. i get to google page where you can push the \"grant access\" button, then oc comes back with no drive attached. no mount.php files gets generated, nothing in the database, nothing in the log file. that's what i get in the ssl_access log file:\r\n```\r\n193.6.210.35 - - [28/jan/2013:15:32:59 +0100] \"post /net-tar2/?app=files_external&getfile=ajax/google.php http/1.1\" 200 237\r\n193.6.210.35 - - [28/jan/2013:15:33:17 +0100] \"get /net-tar2/settings/personal.php?oauth_verifier=xprha2ndmemy_6rax1vphn4t&oauth_token=4%2fis0rhbyrz1tqkjrm1bwv1rgtelpt http/1.1\" 200 13425\r\n193.6.210.35 - - [28/jan/2013:15:33:17 +0100] \"get /net-tar2/?app=files_external&getfile=css%2fsettings.css http/1.1\" 304 -\r\n```\r\ni am in the process of intoducing oc at our local university, so i am doing lots of tests. i wasn't able to mount google drive with 4.5.4, i had success with 4.5.5, now i did a fresh install with 4.5.6 and nothing again. i went to my google account, revoked the access to my own service thinking that was the problem. but that didn't help either.\r\n\r\nwhat can be the problem?"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12794332","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12794332","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12794332,"user":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false},"created_at":"2013-01-28t17:52:05z","updated_at":"2013-01-28t17:52:05z","body":"@torlasz everything should be fixed in this pull request #1031 \r\n\r\nplease test it, i'm still waiting for approval to merge."},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12803526","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12803526","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12803526,"user":{"login":"torlasz","id":2997536,"avatar_url":"https://gravatar.com/avatar/d4184dba791f5d4ca7701cb4d40e21d2?d=https%3a%2f%2fidenticons.github.com%2fa0b69955c7185e378fc6dd2c04979570.png&r=x","gravatar_id":"d4184dba791f5d4ca7701cb4d40e21d2","url":"https://api.github.com/users/torlasz","html_url":"https://github.com/torlasz","followers_url":"https://api.github.com/users/torlasz/followers","following_url":"https://api.github.com/users/torlasz/following{/other_user}","gists_url":"https://api.github.com/users/torlasz/gists{/gist_id}","starred_url":"https://api.github.com/users/torlasz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/torlasz/subscriptions","organizations_url":"https://api.github.com/users/torlasz/orgs","repos_url":"https://api.github.com/users/torlasz/repos","events_url":"https://api.github.com/users/torlasz/events{/privacy}","received_events_url":"https://api.github.com/users/torlasz/received_events","type":"user","site_admin":false},"created_at":"2013-01-28t20:44:12z","updated_at":"2013-01-28t20:44:12z","body":"i'm not familiar with git, that's what i did:\r\ngit clone https://github.com/owncloud/core ./owncloud\r\ngit clone https://github.com/owncloud/apps\r\ngit clone https://github.com/owncloud/3rdparty\r\ncd owncloud\r\ngit branch --track external_storage_ui_feedback remotes/origin/external_storage_ui_feedback\r\ngit checkout external_storage_ui_feedback\r\ni set up the alias, db, permissions etc. enabled the external storage app.\r\nmy problem is that i cannot test attaching gdrive, because selecting anything from drop down boxes doesn't do anything. i cannot even change the language. version is 4.91.2 in config.php. what am i doing wrong?\r\ni would be happy to test it, but i am stuck."},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12804068","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12804068","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12804068,"user":{"login":"torlasz","id":2997536,"avatar_url":"https://gravatar.com/avatar/d4184dba791f5d4ca7701cb4d40e21d2?d=https%3a%2f%2fidenticons.github.com%2fa0b69955c7185e378fc6dd2c04979570.png&r=x","gravatar_id":"d4184dba791f5d4ca7701cb4d40e21d2","url":"https://api.github.com/users/torlasz","html_url":"https://github.com/torlasz","followers_url":"https://api.github.com/users/torlasz/followers","following_url":"https://api.github.com/users/torlasz/following{/other_user}","gists_url":"https://api.github.com/users/torlasz/gists{/gist_id}","starred_url":"https://api.github.com/users/torlasz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/torlasz/subscriptions","organizations_url":"https://api.github.com/users/torlasz/orgs","repos_url":"https://api.github.com/users/torlasz/repos","events_url":"https://api.github.com/users/torlasz/events{/privacy}","received_events_url":"https://api.github.com/users/torlasz/received_events","type":"user","site_admin":false},"created_at":"2013-01-28t20:54:34z","updated_at":"2013-01-28t20:54:34z","body":"sorry, for the trouble, i got it, the 3rd party directory was not set up properly. now attaching a google drive works as expected! "},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12810161","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12810161","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12810161,"user":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false},"created_at":"2013-01-28t22:51:28z","updated_at":"2013-01-28t22:51:28z","body":"@torlasz great! could you add a :+1: to #1031"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/12851901","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-12851901","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":12851901,"user":{"login":"torlasz","id":2997536,"avatar_url":"https://gravatar.com/avatar/d4184dba791f5d4ca7701cb4d40e21d2?d=https%3a%2f%2fidenticons.github.com%2fa0b69955c7185e378fc6dd2c04979570.png&r=x","gravatar_id":"d4184dba791f5d4ca7701cb4d40e21d2","url":"https://api.github.com/users/torlasz","html_url":"https://github.com/torlasz","followers_url":"https://api.github.com/users/torlasz/followers","following_url":"https://api.github.com/users/torlasz/following{/other_user}","gists_url":"https://api.github.com/users/torlasz/gists{/gist_id}","starred_url":"https://api.github.com/users/torlasz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/torlasz/subscriptions","organizations_url":"https://api.github.com/users/torlasz/orgs","repos_url":"https://api.github.com/users/torlasz/repos","events_url":"https://api.github.com/users/torlasz/events{/privacy}","received_events_url":"https://api.github.com/users/torlasz/received_events","type":"user","site_admin":false},"created_at":"2013-01-29t19:11:18z","updated_at":"2013-01-29t19:11:18z","body":"sure. i am really impressed by the speed developers respond here to issues raised! keep up the good work!"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/14391418","html_url":"https://github.com/owncloud/core/issues/1221#issuecomment-14391418","issue_url":"https://api.github.com/repos/owncloud/core/issues/1221","id":14391418,"user":{"login":"torlasz","id":2997536,"avatar_url":"https://gravatar.com/avatar/d4184dba791f5d4ca7701cb4d40e21d2?d=https%3a%2f%2fidenticons.github.com%2fa0b69955c7185e378fc6dd2c04979570.png&r=x","gravatar_id":"d4184dba791f5d4ca7701cb4d40e21d2","url":"https://api.github.com/users/torlasz","html_url":"https://github.com/torlasz","followers_url":"https://api.github.com/users/torlasz/followers","following_url":"https://api.github.com/users/torlasz/following{/other_user}","gists_url":"https://api.github.com/users/torlasz/gists{/gist_id}","starred_url":"https://api.github.com/users/torlasz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/torlasz/subscriptions","organizations_url":"https://api.github.com/users/torlasz/orgs","repos_url":"https://api.github.com/users/torlasz/repos","events_url":"https://api.github.com/users/torlasz/events{/privacy}","received_events_url":"https://api.github.com/users/torlasz/received_events","type":"user","site_admin":false},"created_at":"2013-03-04t17:00:17z","updated_at":"2013-03-04t17:00:17z","body":"since pull request #1031 has been merged into oc 5 rc1 this issue can be regarded as fixed in oc 5"}]